### ISSUE TITLE
Prepare for scpca-docs release: AnnData

### ISSUE THREAD

**Issue by jaclyn-taroni** [AlexsLemonade/scpca-docs#202]
## Steps for a new release of `scpca-docs`

This docs release is associated with the availability of AnnData downloads from the ScPCA Portal

### Preparing for the release
- [x] Does this release require an update to the CHANGELOG because there will be changes to downloadable files? If so, please create an issue tracking the CHANGELOG update and mark it as blocking this issue.
- [x] Are all other issues planned for this release resolved? If any issues are unresolved, mark this issue as blocked by those on ZenHub.
- [x] Optional: If not all changes in `development` are ready to be released, create a feature branch off `main` and cherry pick commits from `development` to that feature branch.
- [x] File a PR from the `development` branch (or the new feature branch) to the `main` branch. This should include all of the changes that will be associated with the next release.
  - [x] If a CHANGELOG entry was required, add the date to the entry's header as part of this PR.

### Creating a release
- [x] On the [releases page](https://github.com/AlexsLemonade/scpca-docs/releases), choose `Draft a new release`.
- [x] In `Choose a tag`, use the current date as the release tag (`YYYY.MM.DD`), then click `Create a new tag: YYYY.MM.DD on publish`.
- [x] Write a description of the major changes in this release. You may want to start with the auto-generated release notes to save time.
- [ ] Optional: If not all issues have been addressed, save a draft to return to later.
- [x] Publish the release!

**Comment by allyhawkins** [comment]
Just noting that I have cherry-picked all AnnData docs related changes to be on `allyhawkins/anndata-release`.  We still need a changelog entry that should be in its own PR, but once that is in, then that branch will be ready for a PR to `main`.
